packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/fresco/ReactOkHttpNetworkFetcher.kt

```diff
@@ -43,6 +44,11 @@ internal class ReactOkHttpNetworkFetcher(private val okHttpClient: OkHttpClient)
         ImageCacheControl.RELOAD -> {
           cacheControlBuilder.noCache()
         }
+        ImageCacheControl.FORCE_CACHE -> {
+          cacheControlBuilder
+            .onlyIfCached()
+            .maxStale(Integer.MAX_VALUE, TimeUnit.SECONDS)
+        }
```

Comment:
Ref: [okhttp/src/main/kotlin/okhttp3/internal/-CacheControlCommon.kt](https://github.com/square/okhttp/blob/f1e6d01aea30844f0ddc7394208ec13d59d5e6f8/okhttp/src/main/kotlin/okhttp3/internal/-CacheControlCommon.kt#L61-L65)

Reply:
Is `onlyIfCached` the correct add here? Per MDN:
>The client indicates that an already-cached response should be returned. If a cache has a stored response, even a stale one, it will be returned. If no cached response is available, a [504 Gateway Timeout](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504) response will be returned.

Reply:
You are right, I was overlooking it when using the reference from `OkHttp3` as that utility has `onlyIfCache` set, but for our logic it doesn't apply. I've removed it in dca3d3a08f9604aa3a6286e43ef03371a3e56732